src/tpm/kmyth_seal_unseal_impl.c

```diff
@@ -151,10 +154,82 @@ int tpm2_kmyth_seal(uint8_t * input,
     return 1;
   }
 
+  // optional argument allowing user to receive a hex dump of the policy digest,
+  // to be used to calculate a second policy for policyOR authorization
+  if (bool_trial_only == 1)
```

Comment:
I looks like kmyth-reseal (and kmyth_seal -g) forces this conditional to be executed.

While on a first pass this prints out a hex string to be used as an input parameter for the 'reseal', I don't understand how it is disabled on the subsequent call to kmyth-reseal where the printed out hex string is passed as an input parameter. In that case, I would not think we want the early termination, but instead produce a new .ski file.

Also, I think that the hex string printed out reflects the entire TPM2_DIGEST struct. The first four hex characters are '0200', which I think reflects the little endian representation of the digest size (0x0020 = 32). I think that we should specify only the hex string for the hash itself as a command line input parameter.

Reply:
The comment about the kmyth-reseal workflow was addressed by creating an issue (#171) so that this can be addressed by a future revision.

The comment about the format in which the hex string is printed out was addressed by changes in commit [5ae6a9a](https://github.com/NationalSecurityAgency/kmyth/pull/166/commits/5ae6a9a7e73aa09ed68443fb5be63a6c32741480).